cls/SourceControl/Git/Extension.cls

```diff
@@ -191,7 +191,12 @@ Method OnSourceMenuContextItem(itemName As %String, menuItemName As %String, ByR
     } elseif ##class(SourceControl.Git.Utils).IsInSourceControl(itemName) {
         set Enabled = $case(menuItemName, "AddToSC":-1,:1)
     } else {
-        set Enabled = $case(menuItemName, "AddToSC":1,:-1)
+        if '$$$defClassDefined(classname) {
```

Comment:
Where is classname coming from? Need to see if InternalName is a .CLS and only do this check in that case.

Reply:
I initially had internal name in there and then deleted it because I abandoned that branch to work off of the one pravin made and then forgot to change classname to internalname 🤦‍♂️

Reply:
Will also need to strip the extension (.CLS)